scripts/report_scratch.js

```diff
@@ -77,18 +77,4 @@ db.hosts.aggregate([{$match:{
     function(result){print(result._id.month+"/1/"+result._id.year+", "+result.total/(60*60*1000));});
 
 
-// average task time and cost for a git and patch versions per project
```

Comment:
what is this file used for?

Reply:
I have no idea. It seems like nothing, so I just deleted it.